detector/test_helper.go

```diff
@@ -0,0 +1,48 @@
+package detector
+
+import (
+	"reflect"
+
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/hcl/hcl/parser"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/hashicorp/hcl/hcl/parser (can't find import: github.com/wata727/tflint/vendor/github.com/hashicorp/hcl/hcl/parser)

<!-- Comment by [SideCI](https://sideci.com) -->